src/mock/meldinger/oppgave-mock.ts

```diff
@@ -95,21 +95,21 @@ function getOppgavetyper(): GsakTemaOppgavetype[] {
 }
 
 function getPrioriteterUtenNormal(): GsakTemaPrioritet[] {
-    return getPrioriteter().filter(it => !it.kode.includes('NORM'));
+    return getPrioriteter().filter((it) => !it.kode.includes('NORM'));
```

Comment:
Lite spørsmål: er dette kun formatering eller har det funksjonell betydning (it =>) vs ((it) =>....)

Reply:
Jepp, vi har oppdatert reglene i Prettier (som brukes for formateringen) for en tid tilbake, så disse endringene kommer ettersom vi endrer på filer som ikke har blitt gjort noe med på en stund. Så kun formattering, ja 👍🏼